README.md

```
@@ -3,482 +3,472 @@
-- [Prerequisites](#prerequisites)
-- [Getting Started](#getting-started)
-- [GitOps Workflow](#gitops-workflow)
-- [EKS Auto Mode](#eks-auto-mode)
-- [Infrastructure Components](#infrastructure-components)
-- [CI/CD Pipeline](#cicd-pipeline)
-- [Monitoring and Observability](#monitoring-and-observability)
-- [Cleanup](#cleanup)
-- [Troubleshooting](#troubleshooting)
-
-## Overview
+- [Quick Start](#-quick-start)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)